pallets/staking/src/lib.rs

```diff
@@ -981,6 +983,7 @@ decl_module! {
             }
 
             // reject a bond which is considered to be _dust_.
+            // Not needed this check as we removes the Exestential deposit concept
```

Comment:
It doesn't really matter, but shouldn't:
`let value = value.min(stash_balance);`
be before:
```
            if value < T::Currency::minimum_balance() {
                Err(Error::<T>::InsufficientValue)?
            }
```

Reply:
No, `T::Currency::minimum_balance()` gives existential deposit while `value.min(stash_balance)` is giving minimum value within those two. So it means we are neglecting who is trying to bond value below the existential deposit. Even if we remove that check it doesn't affect anything because the existential value is 0 in our case.